packages/cozy-harvest-lib/src/services/budget-insight.js

```diff
@@ -65,14 +65,10 @@ const convertBIErrortoKonnectorJobError = error => {
   throw err
 }
 
-export const isBudgetInsightConnector = konnector => {
-  return (
-    konnector.partnership &&
-    konnector.partnership.domain.includes('budget-insight')
-  )
-}
+export const isBudgetInsightConnector = konnector =>
+  konnector?.partnership?.domain === 'budget-insight.com'
```

Comment:
Are we sure that all konnectors have this specific domain? Since before it was `.includes()` it was less trict